core/src/services/dropbox/backend.rs

```diff
@@ -93,8 +94,8 @@ impl Accessor for DropboxBackend {
         }
     }
 
-    async fn read(&self, path: &str, _args: OpRead) -> Result<(RpRead, Self::Reader)> {
-        let resp = self.core.dropbox_get(path).await?;
+    async fn read(&self, path: &str, args: OpRead) -> Result<(RpRead, Self::Reader)> {
+        let resp = self.core.dropbox_get(path, args).await?;
         let status = resp.status();
         match status {
             StatusCode::OK => {
```

Comment:
I'm guessing dropbox may return other status code in this case.

Reply:
I'll try it locally. The log from github action had been a mess.